docs/source/command_line.rst

```diff
@@ -593,6 +593,41 @@ in error messages.
 
     Show absolute paths to files.
 
+.. option:: --ignore-error-codes CODES
+
+    This flag makes mypy ignore all errors with given error codes. Flag accepts
+    error codes as a comma separated list (there should be no spaces after commas).
```

Comment:
It might be better for usability if spaces around commas were ignored. Error codes can't have spaces.

Grammar nit: "comma separated" -> "comma-separated" (we use the latter spelling elsewhere)